editor/src/components/canvas/canvas-wrapper-component.tsx

```diff
@@ -65,6 +65,7 @@ export const CanvasWrapperComponent = betterReactMemo(
           // ^ prevents Monaco from pushing the inspector out
         }}
       >
+        {/* <FloatingInsertMenu /> */}
```

Comment:
Don't forget to uncomment at least one of these!

Reply:
actually, removed both of these. The whole menu needed to move file, otherwise nothing doing with the z-index